lte/cloud/go/services/subscriberdb/servicers/southbound/subscriberdb_servicer.go

```diff
@@ -249,7 +271,14 @@ func (s *subscriberdbServicer) getSubscribersChangeset(networkID string, clientD
 	if err != nil {
 		return true, nil, nil, err
 	}
-	return false, renewed, deleted, nil
+	cloudflag := GetCloudSubscriberDbEnabled(ctx)
+	if cloudflag {
+		glog.V(2).Infof("Cloud Authentication enabled, not streaming subscriber data")
+		return false, nil, nil, nil
```

Comment:
Move it up to the beginning of the function. There is no reason to waste resources on building the changeset if it'll get discarded anyway.